src/style.css

```diff
@@ -1,3 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+
+  --main-color: rgba(0, 172, 220, 1);
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  place-content: stretch space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  padding: 10px 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  font-size: 12px;
+  font-weight: 500;
+  display: flex;
+  text-transform: uppercase;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
```

Comment:
please, order your css properties in a [better way](https://9elements.com/css-rule-order/)